### PR TITLE
Add VCS Support

### PR DESCRIPTION
**Pull request by TG1999**
Signed-off-by: TG1999 <tushar.goel.dav@gmail.com>

### CODE REVIEW

File: fetchcode/vcs/__init__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Expose a limited set of classes and functions so callers outside of
```
Comment (pombredanne) [inline fetchcode/vcs/__init__.py:1]: This should be a doctring for the module.

File: tests/test_vcs.py [hunk 2]
```diff
@@ -17,10 +17,26 @@
 from unittest import mock
 
 from fetchcode.vcs import fetch_via_vcs
+from fetchcode.vcs.pip._internal.vcs import vcs
 
 
-def test_fetch_with_git_url_returns_a_response():
-    url = 'git+http://github.com/jamesor/mongoose-versioner'
+def obtain(dest, url):
+    pass
```
Comment (majurg) [inline tests/test_vcs.py:24]: Just for my knowledge, what does this function do?

Comment (TG1999) [reply tests/test_vcs.py:24]: It installs the given VCS Url's content at Destination.

**Comment by TG1999** [comment]
Yeaah sure, I will add more tests now :)

**Comment by pombredanne** [comment]
All good. Merging!

**Pull request merged**